src/Engine/Data/Material.hpp

```diff
@@ -53,6 +53,9 @@ class RA_ENGINE_API Material : public Data::ShaderParameterProvider
      */
     inline void setMaterialName( std::string newName );
 
+    /// Load the material parameter description
+    static void loadMetaData( const std::string& basename, nlohmann::json& destination );
```

Comment:
Should this one moved to ParameterSeetEdit(ing)Interface ?

Reply:
I don't think so. MetaData are a per-class informations, so a static method in each class.